Package.swift

```diff
@@ -75,6 +77,9 @@ let package = Package(
                 "buf.work.yaml",
                 "proto",
                 "README.md",
+            ],
+            swiftSettings: [
+                .unsafeFlags(["-package-name", packageName])
```

Comment:
Have we confirmed that this is not going to cause issues? I have heard and experienced at some point in the past that using unsafe flags causes issues with downstream consumers of the dependency.

Reply:
Given the Eliza SwiftPM app builds fine with the `Connect` package I think it's okay? Do you know what else we should check? I'm not familiar with those issues

Reply:
@rebello95 could we try archiving the Eliza app as a Release IPA? I think that would tell us.

Reply:
Yep, archiving seemed to work fine:
<img width="1136" alt="Screenshot 2023-11-13 at 3 45 27 PM" src="https://github.com/connectrpc/connect-swift/assets/2643476/9065726f-72fe-4351-af2a-4a46e99d407c">

Reply:
Seems you were right 🙃 #217